exercises/rational-numbers/src/test/java/RationalTest.java

```diff
@@ -0,0 +1,260 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RationalTest {
+
+    // Helper methods
+
+    private static final double DOUBLE_EQUALITY_TOLERANCE = 1e-15;
```

Comment:
This seems like a sensible value for a tolerance.

I haven't run this locally yet - is the tolerance necessary? (I can see why it might be) :slightly_smiling_face:

Reply:
It may be, depending on the algorithm used to compute `double pow(double x)` or `double exp(double base)`. I included it because the complex-numbers project has it, and I thought it might be needed for these methods.

Reply:
That makes sense to me - thanks :slightly_smiling_face: